content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "DBT models for CHT Applications"
+linkTitle: "DBT Models"
+weight: 2
+description: >
+  Guide for building DBT models for CHT Applications
+relatedContent: >
+  core/overview/db-schema
+  apps/reference/app-settings/hierarchy
+---
+
+## Overview
+
+CHT Sync copies data from CouchDB to a relational database; it inititally stores the document data from CouchDB in a jsonb column in a single table. This is not possible to query for analytics, so it uses DBT to take the document data and convert it to a relational database format.
+
+The CHT Pipeline repository defines a DBT project, which contains model files for the data schema described [here]({{< ref "core/overview/db-schema" >}}).
+Forms may be specific to each CHT application; to analyze data from form responses to these custom forms, additional models will need to be developed.
+One additional model will be needed for each form, and for any aggreagations, dashboards, or reuseable views which use those form responses as input.
+If using the [configurable contact hierarchy]({{< ref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}), it may also be useful to add models for other contact types.
+
+## Setup
+
+CHT Sync uses a [DBT package](https://docs.getdbt.com/docs/build/packages) to run models, which is configured by providing a URL to a github repository.
+If it is necessary for application specific models to be private, the [cht-pipeline repository](https://github.com/medic/cht-pipeline/) can be forked into a private repository.
+If they can be public, a branch of the public repository can be created.
+
+The url is set in the `CHT_PIPELINE_BRANCH_URL` environment variable, either in .env if using docker compose, or in values.yaml if useing kubernetes.
+
+When models are changed, CHT Sync needs to be restarted for the change to take effect; when it restarts, it automatically downloads the latest version of cht pipeline and applies than changes; for models materialized as views, they are are updated instantly, for models materialized as incmrenetal tables, the table is dropped and will be unavailable until it is rebuilt from scratch.
```

Comment:
It might be worth it to add a sentence discouraging editing of the base models given how long it would take to rebuild the tables if there is already a lot of data in the database.